_board/Seeed_XIAO_nRF52840_Sense.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: download
+board_id: "Seeed_XIAO_nRF52840_Sense"
+title: "Seeed XIAO nRF52840 Sense Download"
+name: "Seeed XIAO nRF52840 Sense"
+manufacturer: "SEEED"
+board_url: "https://www.seeedstudio.com/Seeed-XIAO-BLE-Sense-nRF52840-p-5253.html"
+board_image: "Seeed_XIAO_nRF52840_Sense.jpg"
+date_added: 2021-12-29
+downloads_display: true
+
+
+features:
+  - Bluetooth/BTLE
+  - Battery Charging
+  - USB-C
+
+---
+
+Seeed XIAO BLE Sense has equipped a powerful Nordic nRF52840 MCU which is designed in a Bluetooth 5.0 module, built around 32-bit ARM® Cortex™-M4 CPU with Floating-Point Unit(FPU) operating at 64Mhz. With the capabilities of wireless connection, it still remains the Seeed XIAO series classic form-factor of small and exquisite which can be used for wearable devices and Internet of Things projects. Furthermore, it only costs 5 μA in the deep sleep model and it supports battery charge management because of the BQ25101 chip. 
+In addition, it carries two extra onboard sensors. One of them is a digital microphone created through Pulse Density Modulation(PDM) module on the nRF52840 chip. It can receive audio data in real-time which allows it can be used for audio recognition. The board not only receives audio data but also can speak through the MSM261D3526H1CPM microphone. The other one is 6-axis Inertial Measurement Unit(IMU) which can be applied in TinyML projects like gesture recognition. These onboard sensors provide a great convenience for users and the ultra-small size feature of the board has still remained. 
+Seeed XIAO BLE Sense contains a wealthy interface. The first thing to note is that the Near Field Communication(NFC) is functional on the board. Secondly, there is a tiny and elegant reset button on one side of the Type-C interface. On the other side, it is designed in a three-in-one LED along with a power LED. There are 11 digital i/o that can be used as PWM pins and 6 analog i/o that can be used as ADC pins. It supports UART, IIC, and SPI all three common serial ports. Same as Seeed XIAO RP2040, it has an onboard 2 MB flash which means it can also be programmed by Arduino, MicroPython, CircuitPython, or other program languages. 
+
+
+This board hasn't been fully documented yet. Please make a pull request adding more info to this file.
+
+The description should be written to inform a CircuitPython user what makes the board unique and link to relevant info about it.
```

Comment:
I'm guessing you didn't mean to leave these lines.

Reply:
Oops... I didn't realize it...